src/main/java/com/vint/shop/domain/Roles.java

```diff
@@ -0,0 +1,68 @@
+package com.vint.shop.domain;
+
+import javax.persistence.*;
+import java.util.Objects;
+
+@Entity
+@Table (name = "roles")
+public class Roles {
```

Comment:
Class name should be in the singular - `Role`

Reply:
made